astropy/utils/introspection.py

```diff
@@ -102,16 +100,11 @@ def resolve_name(name, *additional_parts):
     return ret
 
 
-@deprecated_renamed_argument("version_path", None, "5.0")
-def minversion(module, version, inclusive=True, version_path="__version__"):
+def minversion(module, version, inclusive=True):
     """
     Returns `True` if the specified Python module satisfies a minimum version
     requirement, and `False` if not.
 
-    .. deprecated::
-        ``version_path`` is not used anymore and is deprecated in
-        ``astropy`` 5.0.
```

Comment:
Should the docstring record that an argument was removed in 6.0?

Reply:
yes I think we just remove this mention.